src/Microsoft.ML.AutoML/Assembly.cs

```diff
@@ -3,11 +3,8 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Runtime.CompilerServices;
-using Microsoft.ML;
 
 [assembly: InternalsVisibleTo("Microsoft.ML.AutoML.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a34928e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4")]
-[assembly: InternalsVisibleTo("mlnet, PublicKey=00240000048000009400000006020000002400005253413100040000010001004b86c4cb78549b34bab61a3b1800e23bfeb5b3ec390074041536a7e3cbd97f5f04cf0f857155a8928eaa29ebfd11cfbbad3ba70efea7bda3226c6a8d370a4cd303f714486b6ebc225985a638471e6ef571cc92a4613c00b8fa65d61ccee0cbe5f36330c9a01f4183559f1bef24cc2917c6d913e3a541333a1d05d9bed22b38cb")]
-[assembly: InternalsVisibleTo("mlnet.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a34928e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4")]
 [assembly: InternalsVisibleTo("Benchmark, PublicKey=00240000048000009400000006020000002400005253413100040000010001004b86c4cb78549b34bab61a3b1800e23bfeb5b3ec390074041536a7e3cbd97f5f04cf0f857155a8928eaa29ebfd11cfbbad3ba70efea7bda3226c6a8d370a4cd303f714486b6ebc225985a638471e6ef571cc92a4613c00b8fa65d61ccee0cbe5f36330c9a01f4183559f1bef24cc2917c6d913e3a541333a1d05d9bed22b38cb")]
-
-[assembly: WantsToBeBestFriends]
```

Comment:
You should keep this too.

Reply:
I had to remove this because it would cause build failures:

```
...
CodeGenerator\CSharp\TransformGenerators.cs(219,35): error MSML_NoBestFriendInternal: Access of 'PipelineNode' is a cross assembly internal reference, and the declaring assembly wants these accesses to be on something with the attribute Microsoft.ML.BestFriendAttribute. [F:\workspace.4\_work\1\s\src\Microsoft.ML.CodeGenerator\Microsoft.ML.CodeGenerator.csproj]
CodeGenerator\CSharp\TransformGenerators.cs(254,32): error MSML_NoBestFriendInternal: Access of 'PipelineNode' is a cross assembly internal reference, and the declaring assembly wants these accesses to be on something with the attribute Microsoft.ML.BestFriendAttribute. [F:\workspace.4\_work\1\s\src\Microsoft.ML.CodeGenerator\Microsoft.ML.CodeGenerator.csproj]
CodeGenerator\CSharp\TransformGenerators.cs(277,31): error MSML_NoBestFriendInternal: Access of 'PipelineNode' is a cross assembly internal reference, and the declaring assembly wants these accesses to be on something with the attribute Microsoft.ML.BestFriendAttribute. [F:\workspace.4\_work\1\s\src\Microsoft.ML.CodeGenerator\Microsoft.ML.CodeGenerator.csproj]
CodeGenerator\CSharp\TransformGenerators.cs(312,34): error MSML_NoBestFriendInternal: Access of 'PipelineNode' is a cross assembly internal reference, and the declaring assembly wants these accesses to be on something with the attribute Microsoft.ML.BestFriendAttribute. [F:\workspace.4\_work\1\s\src\Microsoft.ML.CodeGenerator\Microsoft.ML.CodeGenerator.csproj]
...
Build FAILED.
```